yarn-project/aztec-cli/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "@aztec/aztec-cli",
-  "version": "0.0.0",
+  "name": "@aztec/azti",
```

Comment:
![Uploading image.png…]()

Reply:
What's 'azti'?

Reply:
name we came up with for the CLI with @Maddiaa0 & @Cooper-Kunz 
Thinking about it the tool maybe could be named `@aztec/cli` on npm for better discoverability, and just have `azti` as the command being used when install